docs/code-search/working/saved_searches.mdx

```diff
@@ -1,50 +1,38 @@
-# Saved Search
+# Saved Searches
 
 <p className="subtitle">This page provides docs about how Saved Searches work with Sourcegraph.</p>
 
-Saved Search lets you save and describe search queries so you can easily find and use them again later. You can create a saved search for anything, including diffs and commits across all branches of your repositories.
+Saved Searches lets you easily save and describe search queries to find and use them again later. You can create a saved search for anything, including diffs and commits across all branches of your repositories.
 
-If you want Sourcegraph to monitor a search query and send notifications when there are new results, use [code monitoring](/code_monitoring/).
+Saved Searches functionality is available to both Free and Enterprise Code Search users. To access or create new Saved Searches in the Sourcegraph web app, click the **Tools > Saved Searches** in the top navigation bar.
 
-## Creating saved searches
-
-A saved search consists of a description and a query, both of which you can define and edit.
+![saved-searhces](https://storage.googleapis.com/sourcegraph-assets/Docs/saved-searches-0824.png)
 
-Saved searches can be associated with a user or an org:
+## Creating saved searches
 
-- User saved searches are only visible to (and editable by) the user that created them.
-- Org saved searches are visible to (and editable by) all members of the org.
+To create a new saved search:
 
-### User saved searches
+- Go to the Saved Searches section and click the **New saved search** button
+- Fill out the description field and define the syntax for the search query
+- While writing the query syntax, ensure to include the `patternType:` field
 
-To create a User saved search:
+A `patternType:` filter is required in the query for all saved searches. `patternType` can be `keyword`, `standard`, `literal`, or `regexp`. You cannot create a saved search without defining the `patternType:` field.
 
-1. Go to **User menu > Saved searches** in the top navigation bar.
-1. Press the **+ Add new search** button.
-1. In the **Query** field, type in the components of the search query.
-1. In the **Description** field, type in a human-readable description for your saved search.
-1. Click **Create**. The saved search is created, and you can see the number of results.
+![create-new-saved-search](https://storage.googleapis.com/sourcegraph-assets/Docs/create-new-saved-search-0824.png)
 
-Alternatively, to create a saved search from a search you've already run:
+Enable the checkbox for **Draft** if you don't want other users to use your saved search. This is useful for testing the query syntax before sharing it with others. Once done, click the **Create saved search** button to be redirected to the Saved Searches page.
```

Comment:
Don't use "syntax" here. Just call the noun a "search" or "query"

For Draft, I'd recommend something like "... if you want other users to know that it's not ready and shouldn't be relied upon". They can still see drafts, it's just a visual indicator.